README.md

```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
 
 
 [helm]: https://helm.sh/
+
+# Deploy RHTAP
+Follow the below steps to deploy RHTAP on Openshift cluster.
+
+1.Clone the repositry.Run the command 'make` from the rhtap-cli directory.
+This will create a bin directory.
+
+2.Edit the config.yaml file for select or deselect the components from installation.
```

Comment:
Could you create a link to the repository file `config.yaml`? There are examples before that you can copy and paste.